bot/moderation/moderation.py

```diff
@@ -22,17 +26,540 @@ def __init__(self, bot):
         Args:
             bot (discord.ext.commands.Bot): The bot for which this cog should be enabled.
         """
-        self.bot = bot
         self._db_connector = DatabaseConnector(constants.DB_FILE_PATH, constants.DB_INIT_SCRIPT)
+        self.scheduler = AsyncIOScheduler(job_defaults={'misfire_grace_time': 24*60*60},
```

Comment:
If a job couldnt be fired for the duration of the "misfire_grace_time" (maybe because the bot is simply not running) the scheduler won't try to fire it again. This is just to prevent some kind of pile up regarding tasks which would then all be executed at once. I chose 1 day because I think most of the time we would realize that something is wrong beforehand so it shouldn't really matter.